cogs/numb.py

```diff
@@ -0,0 +1,63 @@
+'''This is a guess the number game'''
+import random
+import asyncio
+from discord.ext import commands
+
+
+GuessesTaken = 0
+
+class Guess:
+	'''this is a number guessing game'''
+	def __init__(self, bot):
+		self.bot = bot
+	@commands.command(name="numguess", pass_context=True)
+	async def mycom(self, context):
+		'''this will start a number guessing game'''
+
+		await self.bot.reply("how hard would you like the number game to be? easy/med/hard")
+		mode = self.bot.wait_for_message(channel=context.message.channel, timeout=30)
+		if mode == 'easy':
+			number = random.randint(1, 10)
+
+		elif mode == 'med':
+			number = random.randint(1,50)
+
+		elif mode == 'hard':
+			number = random.randint(1,100)
+
+		#easter egg
+		elif mode == 'no u':
+			number = randome.randint(1,420)
+			await self.bot.say('no u')
+
+		else:
+			await self.bot.say('type again')
+			return
```

Comment:
FYI, the user doesn't get a chance to retype.  They have to reinvoke the original command again.

Reply:
shhhhh.....
no one needs to know.......